### DISCUSSION TITLE
tools/jekyll-blog/build-a-blog/

### DISCUSSION THREAD

**Discussion by giscus** [dev-ujin/dev-ujin.github.io#7] [Comments]
# tools/jekyll-blog/build-a-blog/

💎Jekyll + Github Pages로 개발 블로그 만들기 / Minimal-Mistakes 테마 커스터마이징하기💎

https://dev-ujin.github.io/tools/jekyll-blog/build-a-blog/

**Comment by brightennote** [comment]
블로그 만들기에 도움이 많이 됬어요! 감사합니다~

**Reply by devyoujin** [reply]
감사합니다 :))
